content/en/apps/guides/debugging/obtaining-logs.md

```diff
@@ -1,27 +1,63 @@
 ---
 title: "Obtaining Logs"
 linkTitle: "Obtaining Logs"
-weight: 
+weight:
 description: >
-  How to obtain logs from an instance
+  How to obtain server and client logs
 relatedContent: >
-  
+
 ---
 
 There are many places where useful logs reside. This details all those places, and the easiest way to get a hold of them.
 
-## Browser logs
+## Client logs
 
-To check if there are relevant logs open up the developer console. The shortcut is probably COMMAND+ALT+I on MacOS, or CTRL+ALT+I on Linux and Windows. Click the console tab and copy out any errors or logging that you think is relevant.
+### On a laptop or desktop
 
-## Server logs
+To check if there are relevant logs open up the developer console in your browser. The shortcut is probably COMMAND+ALT+I on MacOS, or CTRL+ALT+I on Linux and Windows. Click the console tab and copy out any errors or logging that you think is relevant.
+
+### On a phone
+
+There are two types of logs updated by Android devices depending on what information is needed. If you trying to get either of the logs from a device for the first time you need to set it up for USB debugging.
+
+1. Enable USB debugging on the phone. This varies from phone to phone, but here is the [official Android documentation](https://developer.android.com/studio/debug/dev-options#enable).
+2. Connect your phone by USB to the computer.
+3. You will see a popup _"Allow USB debugging. The computer's RSA fingerprint..."_, click _"OK"_. If you don't see the popup when the cable is plugged in, you will see it after the next step.
+
+   ![Allow USB debugging](/apps/guides/debugging/images/allow_usb_debugging.png)
+
+#### Browser logs
+
+These are the same as the desktop browser logs. Now that you've enabled USB debugging on the phone you can access the dev console on the phone via USB from your desktop browser. Follow the [official Android documentation](https://developer.chrome.com/docs/devtools/remote-debugging/webviews/#open_a_webview_in_devtools) to access the console.
```

Comment:
I totally see what you're saying, but the devils advocate in me says, "if these are the same as the desktop browser logs, why do I need them? Don't I already have them on my desktop?".  

So maybe a fix might be...

```suggestion
Now that you've enabled USB debugging on the phone you can access the dev console on the phone via USB from your desktop browser. This allows the same level of debugging and inspection you have on a desktop browser, but from your phone's browser in the CHT app. Follow the [official Android documentation](https://developer.chrome.com/docs/devtools/remote-debugging/webviews/#open_a_webview_in_devtools) to access the console.
```

Still maybe a bit awkward? Hopefully you see what I'm getting et.